Dockerfile.cli

```diff
@@ -0,0 +1,24 @@
+ARG image_tag=latest
+ARG php_version
+FROM elifesciences/journal_assets:${image_tag} AS assets
+FROM elifesciences/journal_composer:${image_tag} AS composer
+FROM elifesciences/php_7.0_cli:${php_version}
+
+ENV PROJECT_FOLDER=/srv/journal
+WORKDIR ${PROJECT_FOLDER}
+
+USER root
+RUN pecl install redis && \
```

Comment:
a reason to avoid `docker-php-ext-install`?

Reply:
Not available that way.

Reply:
I wonder what magic does `docker-php-ext-install` do :man_shrugging:

Reply:
It can only install extensions that come with the PHP source code.